src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -111,14 +111,15 @@ const DashboardAuthContainer = props => {
   )
 }
 
-const RightBoxSection = ({ title, info, initialOpenState, onClick, children }) => {
+const RightBoxSection = ({ title, info, initialOpenState, afterToggle, onClick, children }) => {
```

Comment:
Thinking out loud: I wish `afterToggle` had a different name... something like `afterTitle`. I was confused at first because I read "toggle" as a verb instead of a noun. Probably just an artifact of how the `Collapse` component evolved (I know you're just using the same name that `Collapse` uses).

Reply:
I was also regretting not changing the prop name in #3240 where I changed the placement of this element. I'll change `afterToggle` to `afterTitle`.

Reply:
Changed in https://github.com/DataBiosphere/terra-ui/pull/3250/commits/c67e2ccae491e8dac1033296f48c33af78753f01.